src/main/java/ca/team2706/frc/robot/Robot.java

```diff
@@ -75,9 +78,12 @@ public void robotInit() {
                 null,                                                                      // 1
                 null,                                                                      // 2
                 OI.getInstance().driveCommand,                                             // 3
-                new DriveOffHab(),                                                         // 4
-                new LevelOneCentreHatch(),                                                 // 5
+                OI.getInstance().driveCommand,                                             // 4
```

Comment:
Aren't we going to get rid of the drive command as an autonomous command since it isn't really necessary?

Reply:
Does this imply that if the selector was accidentally set to one of the settings where it's null, that we wouldn't be able to drive?

Reply:
No, the commands listed here are purely autonomous commands, which is why I don't think we need to also have the drive commands here too because all that does is risk them getting cancelled.